tox.ini

```diff
@@ -17,14 +18,14 @@ deps =
     django19: {[django]1.9.x}
     django110: {[django]1.10.x}
     django_master: {[django]master}
+    py26,py27,py32: mock
```

Comment:
Ah okay! But somehow this test wasn't broken? Is it that it breaks when you have djmoney_rates installed?